crud/src/main/java/com/redhat/lightblue/mediator/Mediator.java

```diff
@@ -301,6 +301,7 @@ public Response update(UpdateRequest req) {
                         updateResponse.setNumMatched(0);
                     }
                 }
+                ctx.getHookManager().ensureQueuedHooksSizeNotTooLarge(factory.getMaxResultSetSizeB(), factory.getWarnResultSetSizeB(), req);
```

Comment:
I see, I read that method as only being used in bulk requests for some reason.

Is it worth coupling whether or not the response is checked to how the hooks work? A performance optimization? Why not set both unconditionally?